CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   policy.
   
 ### Removed
+- Remove Getting Started documentation guide.
```

Comment:
This will also result into builders getting a 404 not found if someone is using it. So few suggestions:
1. Add README server setup link to API overview for server responses: [here](https://aws.github.io/amazon-chime-sdk-js/modules/apioverview.html#1c-create-a-meeting-session-configuration).
The link to add would be : https://github.com/aws/amazon-chime-sdk-js#getting-responses-from-your-server-application
2. In CHANGELOG, mention the link to use API overview which covers the development in more details hence removing the Getting Started guide. This builders can come to CHANGELOG and understand the change and also get the updated link.